### ISSUE TITLE
Don't understand WrongSpeed option

### ISSUE THREAD

**Issue by SKullYeR** [JuanIrache/gopro-telemetry#172]
According to the documentation and my personal understanding of it, the value of WrongSpeed must be in [m/s].

So in theory all GPS points with speeds higher then what i assigned WrongSpeed should be gone.

If i set "WrongSpeed: 55" the max speed should be arround 200  km/h, so extremely generous becouse the speed physically cant exceed 130 km/h in this case, all points get cut, except the first one. 

What is it, that i dont understand?

**Comment by JuanIrache** [comment]
It's hard to tell for sure without looking at the data, but the speed this setting refers to is not the recorded speed, but the speed that can be calculated between samples by dividing distance by time. The recorded speed often includes smoothing out of the box, so it has less extreme values.

**Comment by SKullYeR** [comment]
Thank you verry much for the quick answer,

i will look at my data and give this calcualtion a shot to find the couse of this problem.

Due to restrictions i can't provide you with my data but according to my understanding "WrongSpeed: 55" should cut every point with a calculated speed above  55 m/s, am i right?

So if i got this, i can go ahead and look for the problem in my GPS data.

ty :)

**Comment by JuanIrache** [comment]
Yes. That should be it. The code has to make some decisions along the way, though. For example, if you have a group of positions with slow speeds in one area, and another group of positions with slow speeds in a different area, but going from one area to the other without breaking the speed limit is not possible, the smaller group will disappear
